YNABBER hanging after "found $x account" log

Heyhey,

I regularly have the problem that my YNABBER is doing nothing after showing the following logs:

```
2024/01/03 17:31:26 Successfully sent 257 transaction(s) to YNAB. 0 got skipped and 114 failed.
2024/01/03 17:31:26 Run succeeded
2024/01/03 17:31:26 Waiting 5m0s before running again...
2024/01/03 17:36:27 Found 2 accounts
```

The docker container shows no action after 2024/01/03. 

My requisiton status seems to be "LN" according to the persisted json file. 

Any ideas?

@martinohansen I guess you were right:

> 2024/01/14 00:43:50 Found 2 accounts
2024/01/14 00:43:50 Reading transactions from account: DEXXXXXXX
panic: reading: failed to get transactions: Get "https://ob.nordigen.com/api/v2/accounts/9eec0b53-6824-455c-a4bf-bba5a298e4c1/transactions/": net/http: request canceled (Client.Timeout exceeded while awaiting headers)

The next step would be ignoring that error or understand it better?

--- a/cmd/ynabber/main.go
+++ b/cmd/ynabber/main.go
@@ -7,6 +7,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/carlmjohnson/versioninfo"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/martinohansen/ynabber"
 	"github.com/martinohansen/ynabber/reader/nordigen"
@@ -15,6 +16,8 @@ import (
 )
 
 func main() {
+	log.Println("Version:", versioninfo.Short())
+
 	// Read config from env
 	var cfg ynabber.Config
 	err := envconfig.Process("", &cfg)

--- a/go.mod
+++ b/go.mod
@@ -2,7 +2,7 @@ module github.com/martinohansen/ynabber
 
 go 1.21
 
-require github.com/frieser/nordigen-go-lib/v2 v2.1.4
+require github.com/frieser/nordigen-go-lib/v2 v2.1.5
 
 require github.com/kelseyhightower/envconfig v1.4.0
 

--- a/go.mod
+++ b/go.mod
@@ -5,3 +5,5 @@ go 1.21
 require github.com/frieser/nordigen-go-lib/v2 v2.1.4
 
 require github.com/kelseyhightower/envconfig v1.4.0
+
+require github.com/carlmjohnson/versioninfo v0.22.5

